org.eclipse.january/src/org/eclipse/january/dataset/LazyDatasetBase.java

```diff
@@ -232,6 +169,50 @@ public IMetadata getMetadata() {
 		return null;
 	}
 
+	@Override
```

Comment:
True though we need to express that we do not want "raw" implementations of `MetadataType` or check for them.